portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACWorkspace.path

```diff
@@ -54,6 +54,16 @@
 
 <!--WORKSPACE_SETTINGS-->
 
+<tr>
+	<td>FRIENDLY_URL</td>
+	<td>//input[@name='friendlyURL']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FRIENDLY_URL_ADDRESS</td>
+	<td>//div[contains(@class,'modal-container')]//div[contains(@class,'modal-body')]//p[contains(.,'${friendlyURL}')]</td>
```

Comment:
Hey @calvinctvo, when you use a variable in a .path file it's a good idea to start with "key_" that way it's easier for anyone to understand when seeing the variable in a macro that the variable is being used inside a path

`${key_friendlyURL}`